tests/contrib/celery/test_patch.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from nose.tools import ok_
+from ddtrace import Pin
+
+
+class CeleryPatchTest(unittest.TestCase):
+    def test_patch_after_import(self):
```

Comment:
Thanks for testing this code-path. It ensures the current behavior is preserved without any breaking change.